### ISSUE TITLE
prune option

### ISSUE THREAD

**Issue by jgallen23** [docker/app#347]
if a service is removed from the docker-compose, it would be nice if it gets removed from the stack.  I think this is as simple as passing through the --prune option to docker stack.

**Comment by garethr** [comment]
Good idea, this definitely feels like a good default too. Thanks for the suggestion.

**Comment by doanac** [comment]
I'm not sure if I'm seeing a variation of the same thing or if I've got a different issue. Its easy to produce:

1) Create a docker-app with 2 services
2) docker app run it
3) Remove one of the services and publish new version of the docker-app
4) Do a docker-app-update to the new version.

The service you deleted will still be running.

The update command is awesome about *adding* new services. It also doesn't restart my services which haven't changed. So that part is really cool.
